packages/meditrak-server/src/app/constructApiV2.js

```diff
@@ -0,0 +1,320 @@
+/**
```

Comment:
If you locally diff this file against the deleted `addRoutesToApp`, you'll notice very simple changes between the two. I was hoping github would detect it as a move and edit rather than a delete and create.